Index: src/com/company/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.company;\r\n\r\nimport java.util.Scanner;\r\n\r\npublic class Main {\r\n\r\n    public static void main(String[] args) {\r\n\r\n        var sc = new Scanner(System.in);\r\n\r\n        System.out.print(\"Enter a numeric value: \");\r\n        var d1 = sc.nextDouble();\r\n\r\n        System.out.print(\"Enter a numeric value: \");\r\n        var d2 = sc.nextDouble();\r\n\r\n        double result = d1 / d2;\r\n\r\n        System.out.println(\"The answer is \" + result);\r\n    }\r\n\r\n}\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/company/Main.java b/src/com/company/Main.java
--- a/src/com/company/Main.java	(revision 2b6b598b4649042ed6b730d6c87bfd9bef7a096e)
+++ b/src/com/company/Main.java	(date 1658885260117)
@@ -8,16 +8,20 @@
 
         var sc = new Scanner(System.in);
 
-        System.out.print("Enter a numeric value: ");
-        var d1 = sc.nextDouble();
+        double d1 = getaDouble(sc, "Enter numeric value 1: ");
 
-        System.out.print("Enter a numeric value: ");
-        var d2 = sc.nextDouble();
+        double d2 = getaDouble(sc, "Enter numeric value 2: ");
 
         double result = d1 / d2;
 
         System.out.println("The answer is " + result);
     }
 
+    private static double getaDouble(Scanner sc, String prompt) {
+        System.out.print(prompt);
+        var d1 = sc.nextDouble();
+        return d1;
+    }
+
 }
 
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectKey\">\r\n    <option name=\"state\" value=\"project://e2804f05-5315-4fc6-a121-c522a6c26470\" />\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_13\" default=\"true\" project-jdk-name=\"13\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 2b6b598b4649042ed6b730d6c87bfd9bef7a096e)
+++ b/.idea/misc.xml	(date 1658883013454)
@@ -3,7 +3,7 @@
   <component name="ProjectKey">
     <option name="state" value="project://e2804f05-5315-4fc6-a121-c522a6c26470" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" default="true" project-jdk-name="13" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_16" default="true" project-jdk-name="13" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
